unity-renderer/Assets/Tutorial/Scripts/TutorialConfiguration.cs

```diff
@@ -0,0 +1,62 @@
+using Cinemachine;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace DCL.Tutorial
+{
+    [CreateAssetMenu(fileName = "TutorialConfiguration", menuName = "TutorialConfiguration")]
+    public class TutorialConfiguration : ScriptableObject
```

Comment:
```suggestion
    public class TutorialSettings : ScriptableObject
```
I think it's better if we use the word `settings` here, as it's widely used through the project for the cases where we need to inject an object with configuration.

Reply:
You're right. Done in https://github.com/decentraland/unity-renderer/pull/966/commits/989912d60c62b7549e7b7a24639991144b88cbbc. Thanks!